configs/components/oifs/oifs_grib_output_to_restart.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
```

Comment:
Please, indicate the author of the script and provide contact information. I don't have a problem with providing some shell scripts in the `configs`, as long as the configs don't get flooded with them, and as long as we don't have to support them

Reply:
Can someone also add a small comment for under which circumstances this script is needed?

Reply:
@joakimkjellsson I will add some comments in the next commit. If it's more a personal script, we should move it somewhere else.